cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java

```diff
@@ -272,6 +278,46 @@ public void testHierarchy() throws Exception {
                      getUser("readers1"), Action.READ);
   }
 
+  @Test
+  public void testCache() throws Exception {
+    final NamespaceId ns1 = new NamespaceId("ns1");
+    final Principal alice = getUser("alice");
+    final Principal admin1 = getUser("admin1");
+    try {
+      assertUnauthorized(ns1, alice, Action.READ);
+      assertAuthorized(ns1, admin1, Action.ADMIN);
+    } catch (UnauthorizedException ignored) {
+      //
+    }
+    Assert.assertTrue(2 == authorizer.cacheAsMap().size());
```

Comment:
how is this 2 ? because of the unauthorized which makes the entry with false value ?